### ISSUE TITLE
InitialSilenceTimeout error after recognizing a word

### ISSUE THREAD

**Issue by danielwinkler** [Azure-Samples/cognitive-services-speech-sdk#2043]
**Describe the bug**

We are using `StartContinuousRecognition` to recognize speech, often for single words.
For some recordings, we get events showing that the recognizer identifies a words,

```
RECOGNIZING: Text=Präsidentschaftskandidat 
```

But then we receive a NoMatch event on Recognized.

```
NOMATCH: Speech could not be recognized: [InitialSilenceTimeout] NoMatchReason:InitialSilenceTimeout 
CANCELED: Reason=EndOfStream 
Details: Reason:EndOfStream ErrorDetails:<> 
```

ResultId: `b8fec482bc824e03b05baf22fc5f1f35` 

**Expected behavior**

The audio file clearly says the German word `Präsidentschaftskandidat` which is also identified by the recognizer, it should return this word instead of NoMatch.

**Comment by danielwinkler** [comment]
Log file
[speechlog.log](https://github.com/Azure-Samples/cognitive-services-speech-sdk/files/12299309/speechlog.log)

**Comment by glecaros** [comment]
Hey @danielwinkler, thanks for reaching out.

@fmegen, can you please take a look, from what I read in the log the SDK receives a speech.hypothesis with the expected result, and then a speech.phrase message receives an InitialSilenceTimeout

**Comment by danielwinkler** [comment]
Hi @glecaros , @fmegen ,

are there any updates from your side regarding this issue?

Best,
Daniel

**Comment by fmegen** [comment]
hi @danielwinkler 
What you are seeing is a valid response sequence from the service. What happens is:

The service starts recognizing the audio and emits IntermediatePhraseResult messages to the client. These responses pass a display postprocessing step, applying capitalization for example.

At some point the decoder emits a FinalPhraseResult. This is passed to the display postprocessing step, applying some additional postprocessing to the result, in particular, reordering the NBest results provided by the decoder.

At this stage, the rearranged result may start with an empty phrase (for example the audio was very bad), which is then translated to an InitialSilenceTimeout.

Checking your log, I am seeing an empty NBest[0] result, which explains the status. 
But I am wondering why the intermediate recognition is not listed as a non-top NBest result.

Will need to check

**Comment by danielwinkler** [comment]
Dear @fmegen,

Thank you very much for the explanation of the processing pipeline.
I provided the audio file that caused this behavior, the audio and speech is of good quality.

> But I am wondering why the intermediate recognition is not listed as a non-top NBest result.

As this issue has been closed, I would like to ask if there will be any further investigation or this will be just ignored.
We are somewhat relying on the ASR to pick up single words.
